### PR TITLE
Allow toggling the UI with a /toggleui dev command.

### LINKED ISSUE #6779: Option to Enable/Disable UI ingame or in replay viewer for purdy screenshots.
Title says it all :) Clean screenshots of the battlefield should be possible for us supporting plebs so that we may deliver more glory points to OpenRA.

It would be nice to have a hotkey assigned to turn the UI on and off during games and/or during replays. Perhaps an obscure hotkey as not to accidentally hit it during play.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/5368940-option-to-enable-disable-ui-ingame-or-in-replay-viewer-for-purdy-screenshots?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by phrohdoh**
Closes #6779

### CODE REVIEW

**Comment by penev92** [comment]
I really like the idea, but it will be unusable in multiplayer unless cheats are enabled.
This isn't strictly a debug feature and people might want to use it to take great screenshots (especially once we have screenshot capturing implemented).
So I'd rather have this as a Hotkey that toggles the UI (like Generals did).

**Comment by RoosterDragon** [comment]
Ditto regarding penev's suggestion, though we should ensure there's a "safety" of some sort, otherwise if you slam the hotkey by accident and don't know about it you're screwed.

**Comment by penev92** [comment]
`Esc` should be a good enough safety net.

**Comment by Phrohdoh** [comment]
`Esc` pauses the game so if you can work around that in a logical way, be my guest.

File: OpenRA.Mods.Common/Commands/DevCommands.cs [hunk 1]
```diff
@@ -47,6 +47,7 @@ public void WorldLoaded(World w, WorldRenderer wr)
 			register("all", "toggles all cheats and gives you some cash for your trouble.");
 			register("crash", "crashes the game.");
 			register("levelup", "adds a specified number of levels to the selected actors.");
+			register("toggleui", "toggles the ui (disables pausing) for a better view.");
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Commands/DevCommands.cs]: `(disables pausing)`? You can still pause the game. I think you meant `(disables the menu)`?

**Comment by abcdefg30** [comment]
Is it intentional, that the cash trickle sound silences when using the command?

Also this does not disable the entire UI:
toggleui

**Comment by Mailaender** [comment]
Superseded by https://github.com/OpenRA/OpenRA/issues/8582.

**Pull request closed without merging**